frontend/packages/dev-console/src/utils/pipeline-augment.ts

```diff
@@ -399,3 +399,6 @@ export const getResourceModelFromTask = (task: PipelineTask): K8sKind => {
 
   return getResourceModelFromTaskKind(kind);
 };
+
+export const pipelineExists = (pipelineRun: PipelineRun): boolean =>
```

Comment:
this should be `pipelineRunExists`

Reply:
No @debsmita1, this function checks if pipeline reference is availble in a pipelineRun object, maybe we should change it to `pipelineRefExists`.

Reply:
oh yes, `pipelineRefExists` sounds good. Thank you for clarifying